### PR TITLE
2020-02-07

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2020-02-07-nothing-to-show-for-work.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+image:
+image2:
+author: Valentino Urbano
+---
+
+The amount of work you get done on a given time is not linearly related to the time you spend doing it.
+
+There might be the day when you feel really productive and can write 3000 words in one sitting and the day when you can barely get 500 down and even getting there is a struggle.
+
+Today it is one of those latter days. I've written non for 2 hours with nothing tangible to show for it.
```
Comment (MsSlacker) [inline _posts/2020-02-07-nothing-to-show-for-work.md]: Non?

File: _posts/2020-02-07-nothing-to-show-for-work.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Just unformed, incomplete ideas for possible articles. Not even a first draft. Hopefully I can see a few of the ideas from this writing session evolving into complete articles in the coming days, but for others not so much.
+
+You can see it as laying the groundwork for the coming days. Tomorrow will be better.
```
Comment (MsSlacker) [inline _posts/2020-02-07-nothing-to-show-for-work.md:22]: Will be? Such an optimist.

File: _posts/2020-02-07-nothing-to-show-for-work.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Luckily those days balance themselves out in the end and you always come out ahead with more productive days than the rest.
```
Comment (MsSlacker) [inline _posts/2020-02-07-nothing-to-show-for-work.md:24]: Very, very optimistic.

File: _posts/2020-02-07-nothing-to-show-for-work.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+This is obviously not limited to writing. I experience the same with side projects as well. [I've already talked about it before][1], but to recap, generally the difference for me is that for writing I get these swings on a day to day basis with most days being productive and writing more than what I self impose as the baseline. With side projects instead, these swings are longer and usually happen once a year, but lasts they last longer too.
```
Comment (MsSlacker) [inline _posts/2020-02-07-nothing-to-show-for-work.md]: but lasts they last longer too.
...?

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2020-02-07-nothing-to-show-for-work.md (hunk 2 above)
Comment (valeIT) [reply _posts/2020-02-07-nothing-to-show-for-work.md:22]: It sounded better than "It is reasonably likely that tomorrow is going to be slightly better than today due to how poor today's result turned out to be"

File: _posts/2020-02-07-nothing-to-show-for-work.md (hunk 3 above)
Comment (valeIT) [reply _posts/2020-02-07-nothing-to-show-for-work.md:24]: lol

File: _posts/2020-02-07-nothing-to-show-for-work.md (hunk 4 above)
Comment (valeIT) [reply _posts/2020-02-07-nothing-to-show-for-work.md]: lol right

**Comment by valeIT** [comment]
Will mark the other 2 weird comments as solved... If you have anything else feel free to reopen

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**